### ISSUE TITLE
Feature Request : match Synctrayzor theme with syncthing theme

### ISSUE THREAD

**Issue by stevebeauge** [canton7/SyncTrayzor#285]
Syncthing allows switching between Light and Dark themes.

However, Synctrayzor UI does not. This leads to not nice looking UI :

![2016-06-23](https://cloud.githubusercontent.com/assets/184604/16297350/d1e6ae06-3930-11e6-8eb0-32958d49174c.png)

It may be nice to have theming support in Synctrayzor, ideally matching Syncthing color scheme.

Thanks

**Comment by canton7** [comment]
My concern is that Windows applications generally don't style well: you have to do quite a lot of work in order to style all of the different UI elements (more work than styling Syncthing: lots of elements have subtle gradients and animations on them), and it typically ends up looking pretty rubbish, partially because it doesn't fit in with any of the other windows.

There's also the issue that the style would have to be applied before any windows were shown, but we don't know Syncthing's theme until Syncthing has started.

If someone else wants to put the effort in, I'll probably accept it, but it's an awful lot of work for something which I'm fairly sure wouldn't work anyway I'm afraid.

**Comment by stevebeauge** [comment]
I feared you'd answer that. But I understand. Better put effort in features than look, isn't it ?

thanks for your feedback

**Comment by canton7** [comment]
That's not quite what I said :wink: 

I didn't say "It's work that would be better spent on features", I said "It's an awfullu large amount of work which will probably look crap at the end anyway"

**Comment by danielmreck** [comment]
I agree that getting Synctrayzor to automatically match its theme to Syncthing would be tricky, but perhaps it'd be more straightforward to have a manual Synctrayzor setting for a dark mode. Even if we can't easily skin the Windows-provided UI elements (menubar and scroll bars), don't we have control over the color of the Syncthing console pane?

My eyes would definitely would appreciate this enhancement if we can whittle it down to something that won't take too many resources to do.

**Comment by opusforlife2** [comment]
Agreed. It's mainly the console that provides a glaring contrast with dark themed Syncthing. Making that dark would be good enough.

**Comment by xelra** [comment]
I run Windows in dark mode. Every app from Windows explorer to the browser automatically adjust. Even the Syncthing web ui inside SyncTrayzor automatically switches to dark mode.

It would be really great if SyncTrayzor could style itself with the Windows theme.

**Comment by VsevolodGolovanov** [comment]
Even the tooltip is white, I thought it would be dark if it's a native system drawn control.
![image](https://user-images.githubusercontent.com/8070317/196928675-3f5cb930-0570-49e3-a34b-8b3cc8e614da.png)
![image](https://user-images.githubusercontent.com/8070317/196928701-cb78a38d-6d53-43a4-a25e-7465f247f32f.png)
Actually it doesn't have rounded corners either. I guess it is a custom control then?

**Comment by shanto** [comment]
![Screenshot 2024-09-17 035952](https://github.com/user-attachments/assets/d25bdc55-dbd5-4c70-b017-24fbb56ae39f)

[Work in progress](https://github.com/shanto/synctrayzor/). Waiting for some minor issues (look at that scrollbar!) and theme switcher integration.

@canton7, do you want to help or provide any guidance (where to put the switcher)? Also, are you active these days? I have to decide whether I shall send you a PR or build/release myself.
